tests/test_tzinfo.py

```diff
@@ -163,6 +168,16 @@ def test_comparison(self):
         self.assertFalse(tz <= SMALLEST)
         self.assertTrue(tz >= SMALLEST)
 
+        # offset based comparion tests for tzinfo derived classes like datetime.timezone.
+        utcdatetime = self.DT.replace(tzinfo=timezone.utc)
+        self.assertTrue(tz == utcdatetime.tzinfo)
+        estdatetime = self.DT.replace(tzinfo=timezone(-timedelta(hours=5)))
+        self.assertTrue(self.EST == estdatetime.tzinfo)
+        self.assertTrue(tz > estdatetime.tzinfo)
+        if sys.version_info >= (3, 9) and sys.platform == 'linux':
+            with self.assertRaises(TypeError):
+                tz > ZoneInfo('Europe/London')
```

Comment:
Ah, I meant keep the equality test too, I'll add that back and then let's merge!

```suggestion
            self.assertFalse(tz == ZoneInfo('Europe/London'))
            with self.assertRaises(TypeError):
                tz > ZoneInfo('Europe/London')
```